consent-ui/src/app/ais/initiation/transactions/transactions-consent-review/transactions-consent-review.component.html

```diff
@@ -1 +1,31 @@
-<p>transactions-consent-review works!</p>
+<div class="card">
+  <div class="card-body">
+    <h2 class="mb-4 text-left">Access permission for accounts in {{aspspName}}</h2>
+    <p class="text-secondary mb-5 text-left">Review an account access level for <strong>{{finTechName}}</strong></p>
```

Comment:
Its better to avoid styling with html tags

Reply:
What's the reason for it - I want to stress those letters. If not - what is the alternative?

Reply:
as an alternative: <p class="font-weight-bold">Bold text.</p>